src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm_http_cache_purger.xml

```diff
@@ -15,7 +15,7 @@
 
             <tag name="doctrine.event_listener" event="preUpdate" />
             <tag name="doctrine.event_listener" event="onFlush" />
-            <tag name="doctrine.event_listener" event="postFlush" />
+            <tag name="kernel.event_listener" event="kernel.terminate" method="postFlush" />
```

Comment:
oh, wow ok. has your thinking changed since https://github.com/api-platform/core/pull/1286?

Reply:
Hum, you're right, I'll revert this change as it's not related to this PR. (I forgot about this discussion)

Reply:
but the discussion point is similar, i guess.

Reply:
Not really, to me it's a big deal if a cache is not flushed because you'll serve stale data. Not dispatching a Mercure update is less a problem, because it breaks nothing, and it's explicitly stated in the Mercure protocol that the client MUST call the original API if it wants to be sure to have fresh data. So we can use `kernel.terminate` safely for Mercure, I'm still unsure for Varnish.

Reply:
nicely argued.